{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/account/password_reset_form.jinja

```diff
@@ -0,0 +1,8 @@
+{% from 'forms/field.jinja' import field as f %}
```

Comment:
would love to get this using the default form, perhaps means the default form needs to be updated

Reply:
i added back a default.html because it's not deprecated yet so our div.html wasn't being picked up